test/index.test.js

```diff
@@ -255,16 +258,15 @@ describe('BtpPlugin', function () {
         ],
         error: 'invalid auth_token'
       }
-    ].forEach(function ({label, authData, error}) {
+    ].forEach(function ({ label, authData, error }) {
       it(label, async function () {
-        const connect = this.server.connect()
+        this.server.connect().catch(() => {})
```

Comment:
how come this is throwing an error and being suppressed?

Reply:
Seems like on line 447, an error will always be thrown on disconnect.